dashboard/src/shared/Config.ts

```diff
@@ -1,20 +1,15 @@
 import axios from "axios";
 
-interface IConfig {
+export interface IConfig {
```

Comment:
can you add a comment saying that this is the kubeapps configuration?

Reply:
+1 will do